src/plone/api/content.py

```diff
@@ -569,6 +569,36 @@ def get_uuid(obj=None):
     return IUUID(obj)
 
 
+@required_parameters("obj")
+def get_path(obj=None, relative_to_portal=False):
```

Comment:
Given that `obj` is required I would just remove the decorator and the default:
```suggestion
def get_path(obj, relative_to_portal=False):
```

Reply:
got it! will make the change

i have a question though,
could you clarify how exactly to decide whether to use python's default required parameters and when to use the explicit decorator?

Reply:
Sorry, ignore my suggestion!

Given the way the code is structured, using the decorator is the way to go.
I do not like you are wasting CPU cycles, but it is already like that in all the API helpers.